src/main/java/edu/cornell/kfs/fp/batch/xml/DisbursementVoucherPreConferenceRegistrantXml.java

```diff
@@ -0,0 +1,67 @@
+package edu.cornell.kfs.fp.batch.xml;
+
+import javax.xml.bind.annotation.XmlAccessType;
+import javax.xml.bind.annotation.XmlAccessorType;
+import javax.xml.bind.annotation.XmlElement;
+import javax.xml.bind.annotation.XmlRootElement;
+import javax.xml.bind.annotation.adapters.NormalizedStringAdapter;
+import javax.xml.bind.annotation.adapters.XmlJavaTypeAdapter;
+
+import org.apache.commons.lang3.StringUtils;
+import org.kuali.rice.core.api.util.type.KualiDecimal;
+
+import edu.cornell.kfs.sys.xmladapters.KualiDecimalXmlAdapter;
+
+@XmlAccessorType(XmlAccessType.FIELD)
+@XmlRootElement(name = "DisbursementVoucherPreConferenceRegistrant", namespace = StringUtils.EMPTY)
```

Comment:
Should the name be registrant to match the documentation?

Reply:
The annotations of prepaid_registrants and registrant on DisbursementVoucherPrePaidTravelOverviewXml do match the documentation as fas as I can tell, the root element here just needs to be a unique identifier